coniferest/aadforest.py

```diff
@@ -179,11 +194,12 @@ def fit_known(self, data, known_data=None, known_labels=None):
 
         if known_data is None or len(known_data) == 0 or \
                 known_labels is None or len(known_labels) == 0 or \
-                np.all(known_labels == Label.UNKNOWN):
+                all(label == Label.UNKNOWN for label in known_labels):
```

Comment:
I think, now we can leave this line as it was before.

Reply:
I've fixed it